ipfx/nwb_reader.py

```diff
@@ -446,21 +460,15 @@ def create_nwb_reader(nwb_file):
     """
 
     nwb_version = get_nwb_version(nwb_file)
+    nwb_flavor = get_nwb_flavor(nwb_file)
```

Comment:
You don't need to call `get_nwb_flavour` here, but only below in the `elif nwb_version["major"] == 1:` branch.